rocky/reports/report_types/crawl_report/report.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+
+<h3>Crawl information</h3>
+{% if data.hostnames_and_cookies %}
+    <table>
```

Comment:
Can you put an extra div wrapping the table?
`<div class="horizontal-scroll">`

When the values are too long it will overflow the whole page instead of the table. We have to fix this also for DNS and TLS reports.